try.html

```diff
@@ -1126,6 +1126,7 @@ <h2 id="contributors"> Contributors </h2>
     if (target) {
       target.addEventListener('click', makeMarkupDialogListener(tr));
     }
+    tr.querySelector('th').addEventListener('click', makeMarkupDialogListener(tr));
```

Comment:
I recommend setting `tr.querySelector('th')` to a variable with an explicit name.